ajax/libs/react-imgix/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-imgix",
+  "description": "React Component for displaying an image from imgix",
+  "filename": "react-imgix.jsx",
```

Comment:
This file does not exist in the distributed npm package. As the source repo for this PR has been removed, I'll close this PR as nothing more can be done here. Please open a new PR if you wish to still add this library.

```
node_modules/react-imgix/lib
├── HOCs
│   ├── index.js
│   ├── index.js.map
│   ├── shouldComponentUpdateHOC.js
│   └── shouldComponentUpdateHOC.js.map
├── array-findindex.js
├── array-findindex.js.map
├── common.js
├── common.js.map
├── config.js
├── config.js.map
├── constants.js
├── constants.js.map
├── constructUrl.js
├── constructUrl.js.map
├── extractQueryParams.js
├── extractQueryParams.js.map
├── findClosest.js
├── findClosest.js.map
├── index.js
├── index.js.map
├── react-imgix-bg.js
├── react-imgix-bg.js.map
├── react-imgix.js
├── react-imgix.js.map
├── targetWidths.js
└── targetWidths.js.map
```